scripts/build.sh

```diff
@@ -97,9 +97,22 @@ rm -rf ./proto/.git
 # Primary use is TLS between Broker-CLI and Broker Daemon
 #
 #############################################
-KEY_PATH="./certs/broker-rpc-tls.key"
-CERT_PATH="./certs/broker-rpc-tls.cert"
-CSR_PATH="./certs/broker-rpc-csr.csr"
+
+DIRECTORY=~/.sparkswap
```

Comment:
[nit] maybe put this at the top of the file and call it something more specific like `SPARKSWAP_DIRECTORY` or similar?